user/docker.md

```diff
@@ -0,0 +1,122 @@
+---
+title: Using Docker in Builds
+layout: en
+permalink: /user/docker/
+---
+
+<div id="toc"></div>
+
+Travis CI builds can run and build Docker images, and can also push images to
+Docker repositories or other remote storage.
+
+To use Docker add the following settings to your `.travis.yml`:
+
+```
+sudo: required
+
+services:
+  - docker
+```
+
+Then you can add `- docker` commands to your build as shown in the following
+examples.
+
+### Using a Docker Image from a Repository in a Build
+
+This [example repository](https://github.com/travis-ci/docker-sinatra) runs two
+Docker containers built from the same image:
+
+* a Sinatra application
+* the Sinatra application test suite
+
+After specifying in the `.travis.yml` that the worker is using the legacy
```

Comment:
Hey @plaindocs, doesn't `sudo:required` + `- docker` rather mean the worker will be using the new Trusty image instead of the legacy environment?